### DISCUSSION TITLE
The road to generated SDKs with Kiota using Quarkus

### DISCUSSION THREAD

**Discussion by giscus** [quarkusio/quarkus#39036] [Quarkus Blog/Website]
# The road to generated SDKs with Kiota using Quarkus

Quarkus: Supersonic Subatomic Java

https://quarkus.io/blog/quarkus-kiota/

**Comment by josetesan** [comment]
Awesome tool, will greatly improve developer  experience with it.

Btw, check the blog, there are some typos in the pom, like 
`<version>{kiota-java-extra.version}</version>`
instead of 
`<version>${kiota-java-extra.version}</version>`

And for the sake of re-producing , maybe you should add current versions of those libraries to the sample pom.

**Comment by yuokada** [comment]
Thanks for writing the good entry.

Proposal. In my understanding,  as of now, OpenAPI Specification v3.1.0 isn't supported by kiota.
https://github.com/microsoft/kiota/issues/3914

Could you mention about it at this blog post or the plugin page?
